### PR TITLE
travis: test if it can catch an issue - DO NOT MERGE!

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ deploy: setprivileges
 	kubectl create -f deploy/crd.yaml
 	kubectl create -f deploy/operator.yaml
 	# Create the operator resource (kubectl get apps)
-	kubectl create -f deploy/cr.yaml
+	# kubectl create -f deploy/cr.yaml - test only, try to have it failing just to see it travis catches it
```
Comment (alexxa) [inline Makefile]: I would separate comment line with a code one. It's easier to uncomment and use it then.

Comment (tiraboschi) [reply Makefile]: Right, but my aim here was was just to see if travis CI is really able to detect an issue that make the cdi-operator unable to deploy cdi.
We don't really want to merge this or reuse that comment.

Comment (alexxa) [reply Makefile]: @tiraboschi yeah, me bline, read the title only after I submitted a comment, sorry.

Comment (tiraboschi) [reply Makefile]: @alexxa the good news is that it worked as expected!

**Pull request closed without merging**